packages/protobuf/package.json

```diff
@@ -11,41 +11,54 @@
     "build": "tsc --build --verbose",
     "clean": "rm -rfv dist *.tsbuildinfo package penumbra-zone-*.tgz",
     "dev:pack": "tsc-watch --onSuccess \"$npm_execpath pack\"",
+    "gen:cosmos-sdk": "buf generate buf.build/cosmos/cosmos-sdk:e7a85cef453e4b999ad9aff8714ae05f",
+    "gen:ibc": "buf generate buf.build/cosmos/ibc:7ab44ae956a0488ea04e04511efa5f70",
+    "gen:ics23": "buf generate buf.build/cosmos/ics23:55085f7c710a45f58fa09947208eb70b",
+    "gen:penumbra": "buf generate buf.build/penumbra-zone/penumbra:312294d02bf945ffa4c1b1dd7cd91328",
```

Comment:
versions used in core

https://github.com/penumbra-zone/penumbra/blob/main/proto/buf.lock